lambdas/service/app.py

```diff
@@ -617,6 +617,15 @@ def validate_repository_search(params, **validators):
 max_page_size = 1000
 
 
+def validate_catalog(catalog):
+    try:
+        IndexName.validate_catalog_name(catalog)
+    except RequirementError as e:
+        raise BadRequestError(e)
+    if catalog not in config.catalogs:
```

Comment:
else branch

Reply:
There are other `validate` functions that also have omitted `else` branches
https://github.com/DataBiosphere/azul/blob/476645558a28844bbcdb6d1c931ff06face9ad07/lambdas/service/app.py#L684

https://github.com/DataBiosphere/azul/blob/476645558a28844bbcdb6d1c931ff06face9ad07/lambdas/service/app.py#L713

Should these have also had an `else` branch present?  Following https://github.com/DataBiosphere/azul/blame/develop/CONTRIBUTING.rst#L351